src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -123,4 +129,15 @@ private void setPID(@Config(name = "P", defaultValueNumeric = P) double p,
         motor.config_kD(PID_SLOT_0, d);
     }
 
+    /**
+     * Lowers arm at set speed
+     */
+    public void lowerArm() {
+        motor.set(RETRACT_SPEED);
+    }
+
+    public boolean isHittingLimitSwitch() {
+        return bottomLimitSwitch != null ? bottomLimitSwitch.get() : true;
```

Comment:
Do we anticipate creating this subsystem without the bottom limit switch?

Reply:
This is incase the switch is not mechanically implemented